nixos/tests/moinmoin.nix

```
@@ -13,12 +13,20 @@ import ./make-test.nix ({ pkgs, lib, ... }: {
     };
 
   testScript = ''
-    startAll;
+    import re
 
-    $machine->waitForUnit('moin-ExampleWiki.service');
-    $machine->waitForUnit('nginx.service');
-    $machine->waitForFile('/run/moin/ExampleWiki/gunicorn.sock');
-    $machine->succeed('curl -L http://localhost/') =~ /If you have just installed/ or die;
-    $machine->succeed('moin-ExampleWiki account create --name=admin --email=admin@example.com --password=foo 2>&1') =~ /status success/ or die;
+    start_all()
+
+    machine.wait_for_unit("moin-ExampleWiki.service")
+    machine.wait_for_unit("nginx.service")
+    machine.wait_for_file("/run/moin/ExampleWiki/gunicorn.sock")
+
+    page = machine.succeed("curl -L http://localhost/")
+    assert re.search(r"If you have just installed", page)
```

if it is just a plain substring that you need to match, then REs are a big oversided. How about:

```suggestion
    assert "If you have just installed" in machine.succeed("curl -L http://localhost/")
```